priam/src/main/java/com/netflix/priam/backup/SnapshotBackup.java

```diff
@@ -180,11 +190,50 @@ private void notifyObservers() {
     protected void processColumnFamily(String keyspace, String columnFamily, File backupDir) throws Exception {
 
         File snapshotDir = getValidSnapshot(backupDir, snapshotName);
-        // Add files to this dir
-        if (null != snapshotDir)
-            abstractBackupPaths.addAll(upload(snapshotDir, BackupFileType.SNAP));
-        else
+
+        if (snapshotDir == null) {
             logger.warn("{} folder does not contain {} snapshots", backupDir, snapshotName);
+            return;
+        }
+
+        findForgottenFiles(snapshotDir);
```

Comment:
Is a metric the best way to deal with this, is the eventual plan to automate the movement of these files from the data directory to a `lost+found` directory or some such?

Reply:
The eventual plan is to move (configurable). First step was to ensure we have metrics in place before Priam touch any SSTable.

Reply:
Cool, might want to leave that context in a comment somewhere or a github issue or something? For example I think Priam should require a force option to stop if it detects forgotten files ...